app/src/main/res/layout/news_list_row.xml

```diff
@@ -1,34 +1,26 @@
-<LinearLayout  xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:card_view="http://schemas.android.com/apk/res-auto"
     android:layout_height="wrap_content"
-    android:layout_width="match_parent">
+    android:layout_width="match_parent"
+    android:layout_marginStart="4dp"
+    android:layout_marginTop="4dp"
+    android:layout_marginEnd="4dp">
 
-    <androidx.cardview.widget.CardView xmlns:card_view="http://schemas.android.com/apk/res-auto"
+    <androidx.cardview.widget.CardView
         android:id="@+id/card_view"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_gravity="center"
         card_view:cardBackgroundColor="?attr/cardViewBackgroundColor"
-        card_view:cardCornerRadius="3dp"
-        card_view:cardElevation="6dp"
-        card_view:cardUseCompatPadding="true"
-        card_view:contentPaddingLeft="20dp"
-        card_view:contentPaddingRight="20dp"
-        card_view:contentPaddingTop="4dp">
```

Comment:
This padding needs to be replaced with margins and moved to the rootlayout if you want a nice touch feedback